source/lex.tex

```diff
@@ -477,7 +477,7 @@
 characters \tcode{*/}. These comments do not nest.
 \indextext{comment!\tcode{//}}%
 The characters \tcode{//} start a comment, which terminates immediately before the
-next new-line character. If there is a form-feed or a vertical-tab
+next new-line character. If there is a \unicode{0009}{character tabulation} or a \unicode{000b}{line tabulation}
```

Comment:
That's wrong. Form feed is 0x0c .

Reply:
Thanks --- a foolish error well spotted.